src/api/spec/models/attrib_value_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'rails_helper'
+
+RSpec.describe AttribValue, type: :model do
+  describe '#to_s' do
+    let(:attrib) { create(:attrib_with_default_value) }
+
+    context 'without setting a value' do
+      let(:attrib_value) { create(:attrib_value, attrib: attrib) }
+      let(:default_value) { attrib.attrib_type.default_values.first.value }
+
+      before do
+        attrib_value.reload
```

Comment:
Why do you reload the value here? Now you'd have to remember this in every test case, that you have to reload the value to fetch the default value.